src/pages/Transactions/transactionUtils.tsx

```diff
@@ -13,12 +13,22 @@ const pricePerValidator = new BigNumber(PRICE_PER_VALIDATOR);
 const TX_VALUE = pricePerValidator.multipliedBy(1e18).toNumber();
 
 const isUserRejectionError = (error: any) => {
-  if (error.code === 4001) return true; // metamask
+  if (error.code === 4001) return true; // metamask reject
   if (
     error.message ===
     'MetaMask Tx Signature: User denied transaction signature.'
   )
-    return true; // metamask
+    return true; // metamask reject
+  if (
+    error.message ===
+    '​Error: TransportStatusError: Ledger device: Condition of use not satisfied (denied by the user?) (0x6985)'
+  )
+    return true; // ledger reject via metamask
+  if (
+    error.message ===
+    '​Error: TransportError: Failed to sign with Ledger device: U2F OTHER_ERROR'
```

Comment:
This can be reached in at least two ways:
* The user can have no Ledger connected or not click `confirm` in time, in this case I'd like to return `true` so the TX can be retried.
* The user can time out on the ledger device, but later click confirm. In this case, the TX will still go through.

Unfortunately Metamask doesn't distinguish between these cases (in error message or number), and therefore it is unsafe to offer the option to retry in case the user clicks `Accept` after Metamask times out and the TX is submitted (opening up the risk of a double deposit).

Reply:
I've tried with Metamask in English, Dutch, and German (the latter two were clearly google-translated), the errors are always the same. On the ledger side, I was surprised to learn that they only support English for both their mnemonics, and the firmware running on their devices (Ledger Live can run in other languages though).

It is enough to distinguish the cases and making this change actually helped me catch a bug with an invisible unicode character.